frontend/package.json

```diff
@@ -322,17 +321,16 @@
     "sass": "^1.42.1",
     "sass-loader": "^10.1.1",
     "style-loader": "^2.0.0",
-    "thread-loader": "^3.0.4",
+    "thread-loader": "^4.0.4",
     "ts-jest": "21.x",
-    "ts-loader": "^8.4.0",
     "ts-node": "10.9.2",
     "typescript": "4.5.5",
     "umd-compat-loader": "^2.1.2",
-    "webpack": "^4.47.0",
+    "val-loader": "^6.0.0",
+    "webpack": "^5.75.0",
```

Comment:
Is there a problem jumping to 5.96.1 or was 5.75 just the latest version when you started working on the upgrade?

Reply:
5.75 was just what the SDK used

Reply:
OK, we should look at updating the SDK as well, but let's make it a follow up story.

Reply:
The resolution in `console-dynamic-plugin-sdk` has been removed!